### DISCUSSION TITLE
Some questions about architecture

### DISCUSSION THREAD

**Discussion by Sacoder0000** [FPGA-Research/FABulous#250] [Q&A] [answered]
1.What are the differences between different N_terms?
2.RegFile is what we commonly refer to as BRAM?
3.What's the function of ConfigFSM.v in fold Fabic?

**Comment by KelvinChung2000** [comment, marked as answer]
Different tiles have different interfaces along the border. For example, LUT tiles have extra `cin' and `co` compared to others. They are there for completeness. They also added reflectors for signals. But I think there should be a way to avoid different " terms" in a fabric. 

It's not quite BRAM; I would just say it's some register-based storage element. If I recall correctly, the arrangement is 32 4-bit entries, which reassemble something more like a register file. 

It is the state machine for controlling how configuration should happen.
